app/web/global.d.ts

```diff
@@ -0,0 +1,6 @@
+interface Window {
+  ReactNativeWebView?: {
+    injectedObjectJson: () => any;
```

Comment:
I think `any` is not allowed once you run the linting command. But kinda useless in a type anyway since it has no type protection.

Reply:
I believe it’s an arbitrary JSON object. Is there a more restricted type for that?

Reply:
You could do `Record<string, any>` if you want to represent an object where the keys are strings, and the values can be of any type. That would at least make sure it's an object!

Reply:
OK, good idea! Thanks.

Reply:
Well I realized it's actually just a string. Not sure why they call it JSON (maybe on the sending end it needs to be JSON).